config/config.go

```diff
@@ -287,7 +287,7 @@ type TCF2 struct {
 	Purpose8            TCF2Purpose             `mapstructure:"purpose8"`
 	Purpose9            TCF2Purpose             `mapstructure:"purpose9"`
 	Purpose10           TCF2Purpose             `mapstructure:"purpose10"`
-	SpecialPurpose1     TCF2Purpose             `mapstructure:"special_purpose1"`
+	SpecialFeature1     TCF2SpecialFeature      `mapstructure:"special_feature1"`
```

Comment:
Interesting. We were using the wrong terminology?

Reply:
It's not just a matter of the wrong terminology. We are looking at the wrong field in the GVL. The go-gdpr package provides a `vendor.SpecialPurpose` method that checks if a special purpose is defined in the GVL `specialPurposes` field for a vendor. We are currently calling this method but that isn't correct. Instead we should be looking at GVL `specialFeatures` for a vendor. I have a go-gdpr PR out that is currently under review that will add a new method that allows us to access the correct field for a given vendor.

Reply:
That'a concerning. What's the difference between a special purpose and a special feature? The TCF folks could've done better in the naming there.